prefab/dependency-creation/README.md

```diff
@@ -0,0 +1,74 @@
+Creating Prefab Dependency AAR
+===============================
+
+This sample shows how to export native libraries to an AAR
+using the [Prefab] format. Generally speaking, to export native libraries in an AAR 
+with Android Gradle Plugin 4.1+, make the following changes to your Android Library project's [build.gradle]:
+
+* Enable the `prefabPublishing` build feature
+```
+    android.buildFeatures.prefabPublising true
+```
+
+* Declare the libraries and headers you wish to export in the `android.prefab`
+  block
+```
+    android.prefab.mylibrary {
+        headers "src/main/cpp/include"
+    }
+```
+* (Optional)If the exported native modules are NOT used by local Kotlin/Java modules in this AAR,
+  you could temporarily avoid [the unnecessary duplication inside AAR]
+```
+    packagingOptions {
+        exclude("**/libmylibrary.so")
+        exclude("**/libc++_shared.so")
+    }
+```
+This sample builds a trivial library called `mylibrary` (libmylibrary.so), and packs it into 
+mylibrary.aar. From here you could distribute the AAR via Maven center or in other ways.
+For how to use the generated dependency AAR, refer to [the dependency-integration sample] in this repo; for the official documentation about Prefab in AGP, go to [C/C++ dependencies] website.
+
+[Prefab]:https://google.github.io/prefab/
+[build.gradle]:https://github.com/android/ndk-samples/blob/master/prefab/prefab-publishing/mylibrary/build.gradle#L64
+[the dependency-integration sample]:https://github.com/android/ndk-samples/tree/main/prefab/dependency-integration
+
+
+
+**Note** that you could use any other ways to generate the prefab AAR, as long as the AAR contains a
+legal prefab package.
+
```

Comment:
Is this helpful? That's true of everything in the AAR.

Reply:
this helps in the sense that "this is just one example" of how to generate prefab AAR(s), but not the ONLY way to do it.

Reply:
But if we're not actually pointing people at other ways to do it this isn't really helpful. "You can figure out another way to do this on your own" is true of the entire AAR, but we don't call that out elsewhere because it's obvious.

Reply:
If you want to point people at something, https://android.googlesource.com/platform/tools/ndkports/ and vcpkg are other options.